lazyscribe/artifacts/json.py

```diff
@@ -54,20 +57,26 @@ def construct(
         writer_kwargs : dict, optional (default None)
             Keyword arguments for writing an artifact to the filesystem. Provided when an artifact
             is logged to an experiment.
+        version : int, optional (default None)
+            Integer version to be used for versioning artifacts.
```

Comment:
Should `None` case be described here?
And: according to the text bellow, real default is `0`? Can set `0` as default here...

Reply:
Done,